compose/compose-file.md

```diff
@@ -256,9 +256,11 @@ updates.
 - `monitor`: Duration after each task update to monitor for failure `(ns|us|ms|s|m|h)` (default 0s).
 - `max_failure_ratio`: Failure rate to tolerate during an update.
 
+```
```

Comment:
Now that you mention it, it seems to fit quite nicely - Since the general text is indented due to the list, keeping this whitespace keeps it more uniform in general. 

But overall, either way should be fine. Actually having formatting, be it indented or not, makes the examples much more readable.